github/data_source_github_repositories.go

```diff
@@ -39,9 +46,15 @@ func dataSourceGithubRepositoriesRead(d *schema.ResourceData, meta interface{})
 	client := meta.(*Organization).client
 
 	query := d.Get("query").(string)
+	opt := &github.SearchOptions{
```

Comment:
Another option here would have been to pass the sort option into `searchGithubRepositories(client, query, sort)` keeping all the github specific search logic and types together while keeping this function a bit cleaner. WDYT?

Reply:
That definitely makes sense. Did it this way because it was easier for me to read. I also felt like if someone needed to add additional parameters/values, they could just add it to the function.